--- Makefile.orig	2006-11-23 16:20:01.000000000 +0100
+++ Makefile	2006-11-23 16:20:37.000000000 +0100
@@ -8,6 +8,8 @@
 
 CC      = gcc
 STRIP	= strip
+INSTALL = /usr/bin/install
+PREFIX  = /usr/local
 #CFLAGS  = -O2 -g -Wall -Werror
 CFLAGS  = -Os -Wall -Werror -DDISABLE_DOC # -DTS5300
 LDFLAGS = -lpci -lz -static 
@@ -45,6 +47,9 @@
 		rm -f .test.c .test; exit 1)
 	@rm -f .test.c .test
 
+install: $(PROGRAM)
+	$(INSTALL) flashrom $(PREFIX)/bin
+
 .PHONY: all clean distclean dep pciutils
 
 -include .dependencies
