--- Makefile.orig	2007-04-10 12:36:56.000000000 +0200
+++ Makefile	2007-04-10 12:33:55.000000000 +0200
@@ -11,12 +11,12 @@
 INSTALL = /usr/bin/install
 PREFIX  = /usr/local
 #CFLAGS  = -O2 -g -Wall -Werror
-CFLAGS  = -Os -Wall -Werror -DDISABLE_DOC # -DTS5300
+FLASHROM_CFLAGS  = -Os -Wall -Werror -DDISABLE_DOC # -DTS5300
 OS_ARCH	= $(shell uname)
 ifeq ($(OS_ARCH), SunOS)
-LDFLAGS = -lpci -lz
+FLASHROM_LDFLAGS = -lpci -lz
 else
-LDFLAGS = -lpci -lz -static 
+FLASHROM_LDFLAGS = -lpci -lz
 STRIP_ARGS = -s
 endif
 
@@ -29,7 +29,7 @@
 all: pciutils dep $(PROGRAM)
 
 $(PROGRAM): $(OBJS)
-	$(CC) -o $(PROGRAM) $(OBJS) $(LDFLAGS)
+	$(CC) -o $(PROGRAM) $(OBJS) $(FLASHROM_CFLAGS) $(FLASHROM_LDFLAGS)
 	$(STRIP) $(STRIP_ARGS) $(PROGRAM)
 
 clean:
@@ -47,7 +47,7 @@
 		   echo "struct pci_access *pacc;";	   \
 		   echo "int main(int argc, char **argv)"; \
 		   echo "{ pacc = pci_alloc(); return 0; }"; ) > .test.c )
-	@$(CC) $(CFLAGS) .test.c -o .test $(LDFLAGS) &>/dev/null &&	\
+	@$(CC) $(FLASHROM_CFLAGS) .test.c -o .test $(FLASHROM_LDFLAGS) &>/dev/null && \
 		echo "found." || ( echo "not found."; echo;		\
 		echo "Please install pciutils-devel and zlib-devel.";	\
 		echo "See README for more information."; echo;		\
