--- Makefile.orig	2008-02-21 02:55:10.000000000 +0100
+++ Makefile	2008-02-21 02:55:18.000000000 +0100
@@ -28,11 +28,6 @@
 
 all: pciutils dep $(PROGRAM)
 
-# Set the flashrom version string from the highest revision number
-# of the checked out flashrom files.
-SVNDEF := -D'FLASHROM_VERSION="$(shell svnversion -cn . \
-          | sed -e "s/.*://" -e "s/\([0-9]*\).*/\1/")"'
-
 $(PROGRAM): $(OBJS)
 	$(CC) -o $(PROGRAM) $(OBJS) $(LDFLAGS)
 	$(STRIP) $(STRIP_ARGS) $(PROGRAM)
