--- Makefile.orig	2009-05-21 21:40:04.000000000 +0200
+++ Makefile	2009-05-21 21:42:18.000000000 +0200
@@ -40,8 +40,6 @@
 
 # Set the flashrom version string from the highest revision number
 # of the checked out flashrom files.
-SVNDEF := -D'FLASHROM_VERSION="0.9.0-r$(shell svnversion -cn . \
-          | sed -e "s/.*://" -e "s/\([0-9]*\).*/\1/")"'
 
 $(PROGRAM): $(OBJS)
 	$(CC) $(LDFLAGS) -o $(PROGRAM) $(OBJS) $(LIBS)
