Description: Make the build reproducible
Author: Chris Lamb <lamby@debian.org>
Last-Update: 2016-08-21

--- flashrom-0.9.9+r1954.orig/util/getrevision.sh
+++ flashrom-0.9.9+r1954/util/getrevision.sh
@@ -116,7 +116,9 @@ timestamp() {
 	# freebsd	date [-jnu]  [-d dst] [-r seconds] [-f fmt date | [[[[[cc]yy]mm]dd]HH]MM[.ss]] [+format] [...]
 	# dragonflybsd	date [-jnu]  [-d dst] [-r seconds] [-f fmt date | [[[[[cc]yy]mm]dd]HH]MM[.ss]] [+format] [...]
 	# openbsd	date [-aju]  [-d dst] [-r seconds] [+format] [[[[[[cc]yy]mm]dd]HH]MM[.SS]] [...]
-	if svn_is_file_tracked "$2" ; then
+	if [ -n "$SOURCE_DATE_EPOCH" ] ; then
+		t=$(date -u -d "@$SOURCE_DATE_EPOCH" "$1")
+	elif svn_is_file_tracked "$2" ; then
 		if svn_has_local_changes "$2"; then
 			t=$(date -u "$1")
 		else
